package.json

```diff
@@ -4,7 +4,14 @@
   "description": "- Nodejs 7.x\r - npm: discord.js",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "start": "node src/bot.js",
+    "mocha": "mocha",
+    "test": "docker-compose -f docker-compose-test.yml up --build -d && docker-compose exec digbot npm install --only=dev && docker-compose exec digbot npm test && docker-compose down",
```

Comment:
calling itself, update npm test to npm mocha?

Reply:
Whoops :o